secrets/helper/helper.go

```diff
@@ -238,3 +286,17 @@ func InitCloudSecretsManager(secretsConfig *secrets.SecretsManagerConfig) (secre
 
 	return secretsManager, nil
 }
+
+// MakeKOSKSignature creates KOSK signature which prevents rogue attack
+func MakeKOSKSignature(
+	privateKey *bls.PrivateKey, address types.Address, chainID int64, domain []byte) (*bls.Signature, error) {
+	message, err := abi.Encode(
+		[]interface{}{address, big.NewInt(chainID)},
+		abi.MustNewType("tuple(address, uint256)"))
```

Comment:
This type could be moved to per-package variables instead of creating it every time. It is ok if we call this function only once.

Reply:
It is called only when someone wants to generate KOSK. Later on it will be kept in secrets (which kind of bothers me because KOSK is not secret at all :) )

Reply:
I've my doubts here, because:
1. we're using it only here
2. Secrets init is just used once at setup time.

I prefer not to pollute the package vars with it considering 1 and 2